.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: node_js
 node_js:
+  - "stable"
+  - 11
+  - 10
```

Comment:
@nickmcmillan which version of node should be test?

Reply:
When Travis runs automated tests, does it test against each version of node listed here? If that's the case then happy with what you've proposed.

Reply:
Yes it will run all tests and lint using all node versions.